sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsProvider.java

```diff
@@ -1453,6 +1452,17 @@ public static String prettifyCallstack(Throwable e) {
         return prettifiedCallstack;
     }
 
+    private void handleErrors(Throwable throwable, int systemExitCode) {
+        if (throwable instanceof Error) {
+            LOGGER.error("Unexpected error in DiagnosticsProvider.endSpan. ",  throwable);
```

Comment:
Do we still need this since we call `System.err.println` in the next line of code?

Reply:
want to keep the logger logging if they can show properly so for some customers they do not need to look two places to figure out the issue. 
The System.err is a fallback if the logging does not show up.

Do not see a risk here for keeping both.